src/main/java/duke/AddCommand.java

```diff
@@ -0,0 +1,46 @@
+package duke;
+
+/**
+ * Represents a command class that adds a task.
+ *
+ * @author Sherman Ng Wei Sheng
+ */
+public class AddCommand extends Command {
+    private boolean isExit;
```

Comment:
What is the use of the isExit attribute here? Would it be possible to remove the attribute and just return false in isExit?